knack/commands.py

```diff
@@ -99,6 +99,7 @@ def _resolve_default_value_from_config_file(self, arg, overrides):
             logger.info("Configured default '%s' for arg %s", config_value, arg.name)
             overrides.settings['default'] = DefaultStr(config_value)
             overrides.settings['required'] = False
+            overrides.settings['default_value_source'] = 'Configure'
```

Comment:
`Configure` is a verb. Maybe `config`?

```suggestion
            overrides.settings['default_value_source'] = 'config'
```

Reply:
This value will be shown in help message directly. The other two sources are `Default` and `Local Context`, so maybe `Config` is more reasonable ?

Reply:
Update to `Config`.